website/docs/reference/widgets/menu-button.md

```diff
@@ -1,76 +1,89 @@
+<VideoEmbed host="youtube" videoId="tDMAOxZTmxY" title="How to use Menu Button Widget" caption="How to use Menu Button Widget"/>
+
+## Usage
+
+To use the menu buttons widget, simply drag & drop it onto your app canvas from the widget library and customize it using the properties and events available. The widget comes with default styles, but you can also use the styles tab to customize the look and feel of your buttons and menus. 
 
 ## Properties
 
-Properties allow you to edit the widget, connect it with other widgets and customize the user actions.
+The menu buttons widget has several properties that you can use to customize its behavior and appearance. Properties also allow you to connect it with other widgets and customize the user actions. The table below gives an exhaustive list of available properties for the menu button and menu item.
+
+### Menu Button properties
+These properties allow you to perform formatting changes or bind the Menu Button widget to any other widget, query, or JS object. 
 
-### Widget Properties
+| Property            | Type | Description | Code Snippet |
+| ------------------- | ---------------- |------------------- | ---------------- |
+| **Label**        | Formatting/ Binding | This property sets the label of the menu button. It can be a static text value or dynamically bound to a source of data. For example, you might set the label to be the logged-in user’s name. To dynamically bind the text, use the `{{}}` mustache sign. You can also read the value of the label in the code. | `{{<WIDGET_NAME.label}}`|
+| [**Menu Items Source**](#menu-items-source)| Formatting | This property allows you to specify the source for the menu items. You can define a [static](#static) list of items or provide a [dynamic](#dynamic) data source to be displayed as menu items. |                   |
+| **Visible**          | Formatting/Binding | This property controls the visibility of the widget. For more information, see [Visible property details](/reference/widgets#visible)| `{{<WIDGET_NAME.visible}}`|
+| **Disabled**         | Formatting | This property disables the input to the widget. For more information, see [Disabled property details](/reference/widgets#disabled).|                  |
+| **Animate Loading** | Formatting | This property allows you to control a widget’s animation on the page. This can be useful for adding visual feedback to the user when the widget is loading or processing data.|                  |
+| **Compact**         | Formatting | This property decides if the widget is in compact mode. In compact mode, the widget takes up less space on the page and can be suitable to use in tight layouts.|                 
 
-These properties allow you to edit the Menu Button widget. All these properties are present in the property pane of the widget. The following table lists all the widget properties.
+### Menu Items Source properties
```

Comment:
Let this be Menu Item properties.  the 'Source' might be confusing for users

Reply:
Menu Item is only available when a user chooses static source. That's why Menu Items Source is added so that users know that it's applicable to both static as well as dynamic sources.